data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -222,45 +214,35 @@ podman run -d --net host --privileged --name image-customization \
     --secret pull-secret,mode=400 \
     ${CUSTOMIZATION_IMAGE}
 
-podman run -d --net host --privileged --name ironic-conductor \
+podman run -d --net host --privileged --name ironic \
      --restart on-failure \
      --env IRONIC_RAMDISK_SSH_KEY="$IRONIC_RAMDISK_SSH_KEY" \
-     --env MARIADB_PASSWORD=$mariadb_password \
      --env PROVISIONING_INTERFACE=$PROVISIONING_NIC \
      --env OS_CONDUCTOR__HEARTBEAT_TIMEOUT=120 \
-     --env HTTP_BASIC_HTPASSWD=${IRONIC_HTPASSWD} \
+     --env IRONIC_HTPASSWD=${IRONIC_HTPASSWD} \
+     --env INSPECTOR_HTPASSWD=${IRONIC_HTPASSWD} \
      --env IRONIC_KERNEL_PARAMS=${PROVISIONING_IP_OPTIONS} \
-     --entrypoint /bin/runironic-conductor \
+     --entrypoint /bin/runironic \
      -v $AUTH_DIR:/auth:ro \
      -v $IRONIC_SHARED_VOLUME:/shared:z ${IRONIC_IMAGE}
 
-# We need a better way to wait for the DB sync to happen..
-sleep 10
-
 podman run -d --net host --privileged --name ironic-inspector \
      --restart on-failure \
      --env PROVISIONING_INTERFACE=$PROVISIONING_NIC \
-     --env HTTP_BASIC_HTPASSWD=${IRONIC_HTPASSWD} \
+     --env IRONIC_HTPASSWD=${IRONIC_HTPASSWD} \
+     --env INSPECTOR_HTPASSWD=${IRONIC_HTPASSWD} \
      --env IRONIC_KERNEL_PARAMS=${PROVISIONING_IP_OPTIONS} \
      --entrypoint /bin/runironic-inspector \
      -v $AUTH_DIR:/auth:ro \
      -v $IRONIC_SHARED_VOLUME:/shared:z "${IRONIC_IMAGE}"
 
-podman run -d --net host --privileged --name ironic-api \
-     --restart on-failure \
-     --env MARIADB_PASSWORD=$mariadb_password \
-     --env PROVISIONING_INTERFACE=$PROVISIONING_NIC \
-     --env HTTP_BASIC_HTPASSWD=${IRONIC_HTPASSWD} \
-     --entrypoint /bin/runironic-api \
-     -v $AUTH_DIR:/auth:ro \
-     -v $IRONIC_SHARED_VOLUME:/shared:z ${IRONIC_IMAGE}
-
 podman run -d --name ironic-ramdisk-logs \
      --restart on-failure \
      --entrypoint /bin/runlogwatch.sh \
      -v $IRONIC_SHARED_VOLUME:/shared:z ${IRONIC_IMAGE}
 
 # Failure to start image-customization results in a very confusing error
+sleep 10
```

Comment:
Can we do something better than sleep here, e.g wait for the desired status instead?

Reply:
I agree that even though this part could be improved, the sleep was present also before